create_test_files.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import pandas as pd
+
+from tests.test_build_featurizer import INITIALIZE_MODEL_CASES, INITIALIZED_MODEL_TEST_ARRAY
+from pic2vec.build_featurizer import _initialize_model
+from pic2vec import ImageFeaturizer
+
+TEST_DATA_NAME = 'tests/image_featurizer_testing/csv_checking/testing_data.csv'
+
+MODELS = ['squeezenet', 'vgg16', 'vgg19', 'resnet50', 'inceptionv3', 'xception']
```

Comment:
This list of models probably can use some kind of standardized enum :)

Reply:
Very true